### DISCUSSION TITLE
Verbrauchswerte von Shelly Plug S und Sunnyportal weichen stark ab

### DISCUSSION THREAD

**Discussion by KingLuy** [camueller/SmartApplianceEnabler#504] [Q&A] [answered]
Hallo,

ich hab jetzt mal geschaut. Ich habe eine Shelly Plug S an der ich eine Powerbank lade. Im Shelly Portal sehe ich das ich 27,38Wh verbraucht habe. Im Sunny Portal unter Verbraucherbilanz und -steuerung stehen aber 24,95 kWh mit einem Peak von 99,76KW.

Muss bei der Shelly Plug S mit einem Factor gerechnet werden?

Grüße

**Comment by camueller** [comment]
Du solltest prüfen, was der SAE dem SHM an der /semp-Schnittstelle als Verbrauch meldet.

**Comment by KingLuy** [comment]
`This XML file does not appear to have any style information associated with it. The document tree is shown below.
<Device2EM xmlns="http://www.sma.de/communication/schema/SEMP/v1">
<DeviceInfo>
<Identification>
<DeviceId>F-02011985-000000000001-00</DeviceId>
<DeviceName>X30</DeviceName>
<DeviceType>Charger</DeviceType>
<DeviceSerial>1</DeviceSerial>
<DeviceVendor>PowerXtreme</DeviceVendor>
</Identification>
<Characteristics>
<MaxPowerConsumption>100</MaxPowerConsumption>
</Characteristics>
<Capabilities>
<CurrentPower>
<Method>Measurement</Method>
</CurrentPower>
<Timestamps>
<AbsoluteTimestamps>false</AbsoluteTimestamps>
</Timestamps>
<Interruptions>
<InterruptionsAllowed>false</InterruptionsAllowed>
</Interruptions>
<Requests>
<OptionalEnergy>true</OptionalEnergy>
</Requests>
</Capabilities>
</DeviceInfo>
<DeviceStatus>
<DeviceId>F-02011985-000000000001-00</DeviceId>
<EMSignalsAccepted>false</EMSignalsAccepted>
<Status>On</Status>
<PowerConsumption>
<PowerInfo>
<AveragePower>599</AveragePower>
<Timestamp>0</Timestamp>
<AveragingInterval>60</AveragingInterval>
</PowerInfo>
</PowerConsumption>
</DeviceStatus>
<PlanningRequest>
<Timeframe>
<DeviceId>F-02011985-000000000001-00</DeviceId>
<EarliestStart>0</EarliestStart>
<LatestEnd>21574</LatestEnd>
<MinRunningTime>21547</MinRunningTime>
<MaxRunningTime>21548</MaxRunningTime>
</Timeframe>
<Timeframe>
<DeviceId>F-02011985-000000000001-00</DeviceId>
<EarliestStart>77799</EarliestStart>
<LatestEnd>106599</LatestEnd>
<MinRunningTime>21599</MinRunningTime>
<MaxRunningTime>21600</MaxRunningTime>
</Timeframe>
<Timeframe>
<DeviceId>F-02011985-000000000001-00</DeviceId>
<EarliestStart>164199</EarliestStart>
<LatestEnd>192999</LatestEnd>
<MinRunningTime>21599</MinRunningTime>
<MaxRunningTime>21600</MaxRunningTime>
</Timeframe>
</PlanningRequest>
</Device2EM>`

**Reply by KingLuy** [reply]
Hmmm bekomm es nicht schöner formatiert

**Comment by camueller** [comment]
Wenn Leistungswerte im Sunny Portal scheinbar nicht passen, musst **Du** schauen, was in /semp drinsteht.

In Deinem /semp steht 599W - dieser Wert wurde via SHM an das Sunny Portal übermittelt.

**Comment by KingLuy** [comment]
Das liefert die Steckdose 
`{"power":95.18,"overpower":0.00,"is_valid":true,"timestamp":1679657654,"counters":[95.181, 95.203, 95.191],"total":28073}`

**Comment by camueller** [comment]
Was hast Du denn konfiguriert für den Zähler?

**Comment by KingLuy** [comment]
![image](https://user-images.githubusercontent.com/3822777/227499664-107bde35-3b13-4ec5-8d84-203ba478767b.png)

**Comment by camueller** [comment, marked as answer]
Du musst als Parameter "Leistung" (anstatt "Zählerstand") angeben, wenn Du die Leistung ausliest.

**Comment by KingLuy** [comment]
Jetzt sieht es besser aus! Danke
